wp_bak/app/wp_bak.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# Docker:
+# docker build -t colindavey/wp_bak:alpha .
+# docker run -v `pwd`/src:/home/mvp/app/src -v `pwd`/dst:/home/mvp/app/dst -t colindavey/wp_bak:alpha
+
+from datetime import datetime
+from fnmatch import fnmatch
+import logging
+import os
+import shutil
+import sys
+import tarfile
+import time
+
+#############################
+# CONSTANTS
+#############################
+
+SRC_DIR='src/'
+DST_DIR='dst/'
+SHORT_DIR=DST_DIR+'shorts/'
+LONG_DIR=DST_DIR+'longs/'
+
+MINUTE = 60
+HOUR = 60 * MINUTE
+DAY = 24 * HOUR
+WEEK = 7 * DAY
+
+DATE_TIME_FORMAT = "%Y-%m-%d-%H-%M-%S"
+
+# Duration between backups in seconds
+# Actual value
+# SHORT_DURATION = DAY
+# Value for testing
+SHORT_DURATION = 1
+# Delete everything in the shorts folder older than this
+SHORT_DURATION_TO_KEEP = 7 * SHORT_DURATION
+
+LONG_DURATION = 30 * SHORT_DURATION
+# Delete everything in the longs folder older than this
+LONG_DURATION_TO_KEEP = 2 * LONG_DURATION
+
+# Set to a value for testing
+# NUM_REPS = None
+NUM_REPS = 100
+
+logging.basicConfig(level=logging.INFO, stream=sys.stderr)
+log = logging.getLogger(__name__)
+
+#############################
+# FUNCTIONS
+#############################
+
+def mkdir_if_not_exist(dir):
+    """Makes a directory if it doesn't exist.
+
+    Parameters
+    ----------
+    dir : string, name of directory
+        The directory to make.
+    """
+    if not os.path.exists(dir):
+        os.makedirs(dir)
```

Comment:
Totally fine, no need to change but personally I like pathlib. with that it'd be something like:

```
dir = Path('/some/location')
d.mkdir(parents=True, exist_ok=True)
```

Reply:
Done.